app/representers/api/episode_representer.rb

```diff
@@ -12,6 +12,9 @@ class Api::EpisodeRepresenter < Api::BaseRepresenter
   property :prx_uri
   property :published_at
 
+  # combo of title, published_at, and guid at time of first pub
+  property :adzerk_keyword
```

Comment:
Is this supposed to be writeable? Probably not, since it is generated, perhaps add a `writeable: :false` option on this?